components/ide/code/leeway.Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN sudo apt-get update \
     && sudo apt-get clean -y \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GP_CODE_COMMIT ae8dc44299153ca820ac9e13cc9def5bb6baafd1
+ENV GP_CODE_COMMIT 334e0610dd1bb73487812b417b113d6a2b96e1b1
```

Comment:
**question**: I noticed the _Upgrade Subscription_ command is redirecting to `/upgrade-subscription` which does not exist in the preview environment but this should be expected, right? However, are we adding this command also for the self hosted and if yes would this also lead to an error page?

> Sorry, this page couldn't be found 😳
> 404

Reply:
https://github.com/gitpod-io/gitpod/issues/3054